{{cookiecutter.repo_name}}/.github/workflows/CI.yaml

```diff
@@ -0,0 +1,74 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: {{ '${{ matrix.os }}' }}
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest, windows-latest]
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Additional info about the build
+      shell: bash
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+
+{% if cookiecutter.dependency_source != 'Dependencies from pip only (no conda)' %}
+    - name: Setup conda (Ubuntu, macOS)
+      shell: bash
+      if: startsWith(matrix.os, 'macOS') || startsWith(matrix.os, 'ubuntu')
+      run: echo "::set-env name=CONDA_EXE::$CONDA/bin/conda"
+    - name: Setup conda (Windows)
+      shell: bash
+      if: startsWith(matrix.os, 'windows')
+      run: echo "::set-env name=CONDA_EXE::$CONDA/Scripts/conda.exe"
+{% endif %}
+
+{% if cookiecutter.dependency_source == 'Dependencies from pip only (no conda)' %}
+    - name: Install testing dependencies
+      shell: bash
+      run: |
+        python -m pip install -U pytest pytest-cov codecov
+{% else %}
+    - name: Create environment for package
+      shell: bash
+      run: |
+        eval "$(${CONDA_EXE} shell.bash hook)" && conda activate
+        conda info
+        python devtools/scripts/create_conda_env.py -n=test -p={{ '${{ matrix.python-version }}' }} devtools/conda-envs/test_env.yaml
```

Comment:
I like going through ENV variables for these, makes it much easier.

Reply:
Nice! Applied as suggested.